docker_fargate/docker_fargate_stack.py

```diff
@@ -19,7 +19,6 @@
 
 # The name of the environment variable that will hold the secrets
 SECRETS_MANAGER_ENV_NAME = "SECRETS_MANAGER_SECRETS"
-CONTAINER_ENV = "CONTAINER_ENV" # name of env passed from GitHub action
```

Comment:
`ENV` is very general,  `CONTAINER_ENV` seems more appropriate to me in this case